src/mapperReader.ts

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-`use strict`;
+"use strict";
```

Comment:
(is Prettier happy with the double quotes? we use single quotes elsewhere)

Reply:
fixed